src/EA.Weee.Web/Infrastructure/DisplayHelper.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace EA.Weee.Web.Infrastructure
+{
+    using System.Globalization;
+    using EA.Weee.Core.AatfReturn;
+    using EA.Weee.Core.DataReturns;
+
+    public static class DisplayHelper
+    {
+        public static string Reporting = "<b>Reporting on: </b>";
+        public static string Period = "<b>Reporting period: </b>";
+        public static string Tab = "&#09;";
+
+        public static string ReportingOnValue(string name, string number, string quarter)
```

Comment:
Can we create unit tests for these methods.

Reply:
Added